src/Customer.js

```diff
@@ -5,8 +5,9 @@ class Customer {
     this.pastBookings = pastBookings;
     this.upcomingBookings = upcomingBookings;
   }
-  findTotalSpent() {
-    
+  bookRoom(roomNumber, date) {
+    console.log(roomNumber);
```

Comment:
You might want to remove console.logs from your final PR push. Not sure what this function is doing either